services-custom/s3-transfer-manager/src/main/java/software/amazon/awssdk/transfer/s3/model/ResumableFileUpload.java

```diff
@@ -71,15 +71,9 @@ public boolean equals(Object o) {
 
         ResumableFileUpload that = (ResumableFileUpload) o;
 
-        if (totalNumOfParts != that.totalNumOfParts) {
```

Comment:
I see not all the fields/attributes are present in the Equals method , any reason ? Same holds for hash , do we have test case for equals and hashCode ?

Reply:
No reason, I will fix them. Not yet, I'll add tests as I'm finalizing the PR